src/Services/PaymentService.php

```diff
@@ -124,4 +124,19 @@ public function cancel($paymentId)
     {
         return $this->client()->post([], 'queue/payments/{paymentId}/cancel', ['paymentId' => $paymentId]);
     }
+
+    /**
+     * @param array|JsonSerializable|Entities\Payment $payment
+     * @param string $paymentId
+     *
+     * @throws NotFoundException The payment does not exist
+     * @throws NotFoundException The scheduled payment does not exist
+     * @throws GoneException The process is completed but the payment can not be created
+     *
+     * @return Entities\ScheduledPayment|Entities\Payment
+     */
+    public function update($payment, $paymentId)
```

Comment:
To keep consistency, we need to flip arguments. The required URL params have more weight then request body.